app/graphql/inputs.js

```diff
@@ -4,7 +4,6 @@ module.exports = gql`
   input UserInput {
     firstName: String!
     lastName: String!
-    username: String!
```

Comment:
use a deprecation policy

Reply:
username field is deprecated in User model now but deprecation directive is not available in input types